canonical-web-and-design/vanilla-framework
# Update 1.8 refs

## Done

Updated all references to version 1.8.1 as the latest version of the framework to 2.0.0

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- See that 2.0.0 is mentioned in the content

*NOTE* The links will be broken until the framework is deployed
